### PR TITLE
Create some brief instructions

### PR DESCRIPTION
**Pull request by clevernyyyy**

### CODE REVIEW

File: INSTRUCTIONS.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## instructions
+
+Checkout branch `mjs/00-start`.  The objective is to recreate the application on the `master` branch.
```
Comment (kentcdodds) [inline INSTRUCTIONS.md]: Could you mention that all of the steps are under `mjs/xx-`?

File: INSTRUCTIONS.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+
+### further help 
+
+If you'd like to see the differences between the first and second steps, use the `Compare` link found directly below the `Clone or download` green button.
```
Comment (kentcdodds) [inline INSTRUCTIONS.md]: Could you give an example link?

**Comment by clevernyyyy** [comment]
Updated.

**Comment by kentcdodds** [comment]
Perfect! Thanks!

**Pull request merged**